src/olympia/abuse/templates/abuse/emails/CinderActionTargetAppealRemovalAffirmation.txt

```diff
@@ -2,6 +2,10 @@ Hello,
 
 Previously, your {{ type }} was suspended/removed from Mozilla Add-ons, based on a finding that you had violated Mozilla's policies.
 
+{% if has_attachment %}
+An attachment was provided. {% if dev_url %}To respond or view the file, visit {{ dev_url }}.{% endif %}
+
+{% endif %}
```

Comment:
In templates that don't include `.../policies.txt`, have this paragraph following the paragraph that has `{{ manual_reasoning_text }}` in it - i.e. move it further down this file.